### PR TITLE
ARO4 dependency manifest

### PR DESCRIPTION
**Pull request by Makdaam**
### Which issue this PR addresses:
Fixes 8535553

### What this PR does / why we need it:

This PR adds a manifests folder and a manifest for the currently rolled out ARO4 release.

### Test plan for issue:

Communicating with ProdSec if it meets the requirements.

### Is there any documentation that needs to be updated for this PR?

Not at this time.

### CODE REVIEW

**Comment by mjudeikis** [comment]
@Makdaam why do we need this if we don't have any control over it? 
First all of the components are from RHCOS so we can't add/remove anything
Why the security team can't certify RHCOS and make it once and for OSD and ARO?

File: docs/manifests/manifest-7329cd9 [hunk 1]
```diff
@@ -0,0 +1 @@
+openshift-hosted-aro4:7e29cd9/rhcos-44.81.202004250133-0-azure.x86_64
```
Comment (mjudeikis) [inline docs/manifests/manifest-7329cd9:1]: This is used in our code. We could generate it via `go generate` if we move image vars to `pkg/util/version`

**Comment by jim-minter** [comment]
Why do we want to check this into ARO-RP?

**Comment by Makdaam** [comment]
Resolved outside of ARO-RP.

**Pull request closed without merging**